wiki/Tournaments/CMC/2020/en.md

```diff
@@ -0,0 +1,388 @@
+   - Players who ranked #18 - #32 based on their accumulated Qualifier score will be designated as Seed D players.
+   - Players who ranked #33 or lower based on their accumulated Qualifier score will be knocked out from the tournament.
+6. Failure to attend in any of the predetermined Qualifier lobbies will result in an instant elimination from the tournament.
+
+### Group Stage rules
+
+1. In the Group Stage, the 32 previously qualified players will be randomly spread out into 8 groups consisting of 4 different players each based on their seed.
+2. All the players from each group will face each other in a single multiplayer lobby under the "Battle Royale" matchup system as follows:
+   - All players will have to consecutively play all of the six Group Stage maps in the order that it is being written on the [main spreadsheet](https://docs.google.com/spreadsheets/d/1VttS6M8GdWINmMBuHws6AcND0yh3mNGzUiVQwleLP4Q/edit#gid=1467418977).
+   - After the conclusion of each map, the two players that performs the best will receive +2 points while the other two players will receive +1 point.
+3. Players **are not allowed** to ban any maps in the Group Stage.
+4. Players will subsequently be ranked in their respective groups based from the amount of points they accumulate in the room. 
+   - In case of a points tie, all affected players will be ordered by their Group Stage seed.
+
+### Knock-out stage rules
+
+1. Based on each player's Group Stage finishing position, players will then be teamed up into pairs of two where the group winners are set to be matched with the group runners-up and the 3rd place finishers are set to be matched with the 4th place finishers at random.
+2. Teams will compete against each other using the Double Elimination system.
+3. The Double Elimination System works as following:
+   - Teams who lose in the Upper Bracket can still play again on the Lower Bracket.
+   - Teams who lose in the Lower Bracket will be eliminated from the tournament.
+   - In the Grand Finals, the winner of the the Upper Bracket only needs to win a single match against their opponent in order to claim the championship title. The winner of the Lower Bracket, however, needs to win two matches and enforce a Bracket Reset against their opponent in order to claim the championship title.
+4. Teams who can compete in the next round are determined by:
+   - In the Quarterfinals, each team needs to win 5 points in order to win a match. (Best-of-9)
+   - In the Semifinals, each team needs to win 6 points in order to win a match. (Best-of-11)
+   - In both of the Finals weeks, each team needs to win 7 points in order to win a match. (Best-of-13)   
+   - Team(s) who Win By Default.
+   - Whether there are team(s) who got disqualified during the match.
+
+### Match regulations
+
+1. A representative of each team must use the `!roll` command once in #multiplayer in order to determine the banning and picking order.
```

Comment:
#mulitplayer in the in-game chat or the discord server? it would be useful to clarify